src/components/ebay-breadcrumb/test/test.server.js

```diff
@@ -0,0 +1,59 @@
+const expect = require('chai').expect;
+const testUtils = require('../../../common/test-utils/server');
+const mock = require('../mock/index');
+
+describe('should render with basic breadcrumb and ', () => {
+    test('return nav element with input', context => {
+        const $ = testUtils.getCheerio(context.render(mock.basicItems));
```

Comment:
You're still using the same render for the same mock, so you can add a beforeEach to make the code simpler, and more DRY.

```
describe('should render with basic breadcrumb and ', () => {
    let $;

    beforeEach(context => {
        $ = testUtils.getCheerio(context.render(mock.basicItems));
    });

    test('return nav element with input', () => {
        ...
    });

    ...
});
```

Reply:
`beforeEach` will be called for every test in this block. so its a same right?

Reply:
Yes, but think of the ~children~ developer who come after you. 😉

Reply:
Go ahead and leave as-is. We can look into this later.